gnmi_cli_py/py_gnmicli.py

```diff
@@ -435,8 +437,9 @@ def subscribe_start(stub, options, req_iterator):
             break
   except KeyboardInterrupt:
       print("Subscribe Session stopped by user.")
-  except grpc.RpcError as x:
-      print("grpc.RpcError received:\n%s" %x)
+  except grpc.RpcError as err:
+      print("Received an exception from server side and error message is: '{}'.".format(err))
+      raise err
```

Comment:
>err [](http://example.com/codeflow?start=12&length=3)

`raise` without `err` is better.
ref: [https://stackoverflow.com/a/45986945/2514803](https://stackoverflow.com/a/45986945/2514803)

Reply:
Updated.